convert2rhel/main.py

```diff
@@ -85,8 +85,12 @@ def main():
         systeminfo.system_info.resolve_system_info()
         breadcrumbs.breadcrumbs.collect_early_data()
 
+        loggerinst.task("Prepare: Clean yum cache metadata")
+        pkghandler.clean_yum_metadata()
+
         # check the system prior the conversion (possible inhibit)
-        checks.perform_pre_checks()
+        loggerinst.task("Prepare: Perform basic system checks")
+        print(checks.perform_pre_checks())
```

Comment:
```suggestion
```

Looks like debugging that was left in by mistake :-)   removing it should fix the lgtm check.

Reply:
Whoopsie! will remove that asap